forti_token_client.go

```diff
@@ -38,6 +40,8 @@ type fortiTokenClient struct {
 }
 
 func (c *fortiTokenClient) newGetRequest(url string) (*http.Request, error) {
+	http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
+	http.DefaultTransport.(*http.Transport).TLSHandshakeTimeout = 30 * time.Second
```

Comment:
Same here, probably better to make this a flag to avoid magical constants. Something like --https-timeout=30s might be cool.